internal/tracing/handlers/handler.go

```diff
@@ -57,3 +59,78 @@ func BuildOptions(k8s client.Client, mcAddon *addonapiv1alpha1.ManagedClusterAdd
 
 	return resources, nil
 }
+
+func getSecretExporters(otelCol *otelv1alpha1.OpenTelemetryCollector) (map[authentication.Target]string, error) {
```

Comment:
WDYT? `buildExportersSecrets` also works
```suggestion
func buildTargetSecrets(otelCol *otelv1alpha1.OpenTelemetryCollector) (map[authentication.Target]string, error) {
```